vector/src/main/java/im/vector/app/features/media/ImageContentRenderer.kt

```diff
@@ -173,47 +167,9 @@ class ImageContentRenderer @Inject constructor(
                     .load(resolvedUrl)
         }
 
-        req.override(Target.SIZE_ORIGINAL, Target.SIZE_ORIGINAL)
-                .fitCenter()
-                .into(target)
-    }
-
-    fun renderFitTarget(data: Data, mode: Mode, imageView: ImageView, callback: ((Boolean) -> Unit)? = null) {
-        val size = processSize(data, mode)
-
-        // a11y
-        imageView.contentDescription = data.filename
-
-        createGlideRequest(data, mode, imageView, size)
-                .listener(object : RequestListener<Drawable> {
-                    override fun onLoadFailed(
-                            e: GlideException?,
-                            model: Any?,
-                            target: Target<Drawable>?,
-                            isFirstResource: Boolean
-                    ): Boolean {
-                        callback?.invoke(false)
-                        return false
-                    }
-
-                    override fun onResourceReady(
-                            resource: Drawable?,
-                            model: Any?,
-                            target: Target<Drawable>?,
-                            dataSource: DataSource?,
-                            isFirstResource: Boolean
-                    ): Boolean {
-                        callback?.invoke(true)
-                        return false
-                    }
-                })
-                .fitCenter()
-                .into(imageView)
+        req.fitCenter().into(target)
```

Comment:
to double check, this line is the fix? (everything else is removed unused code)

it's interesting that we were overriding the size to be non scaled instead of the default behaviour of scaling to match the `View`

Reply:
(I would have kept the previous formatting, better for git diff, or future change, but this is really a minor remark
```kotlin
req
    .fitCenter()
    .into(target)
```
)

Reply:
Happy to make the change 👍